packages/libconfig.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Libconfig < Package
+  description 'C/C++ Configuration File Library.'
+  homepage 'https://hyperrealm.github.io/libconfig/'
+  version '1.7.2'
+  source_url 'https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.2.tar.gz'
+  source_sha256 '7c3c7a9c73ff3302084386e96f903eb62ce06953bb1666235fac74363a16fad9'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  def self.build
+    system './configure --prefix=/usr/local'
```

Comment:
Please use `#{CREW_PREFIX}` instead of `/usr/local` and enclose in double quotes.  Also, if there are any library files installed in `/usr/local/lib(64)`, those should be abstracted with `--libdir=#{CREW_LIB_PREFIX}`.